src/client/selectors.js

```diff
@@ -12,4 +16,63 @@ export const selectedStockSelector = createSelector(
 export const companyNameSelector = createSelector(
   selectedStockSelector,
   selectedStock => selectedStock.companyName
+);
+
+export const stockOverviewDataSelector = createSelector(
+  selectedStockSelector,
+  selectedStock => selectedStock.stockOverviewData
+);
+
+export const currentPriceSelector = createSelector(
+  stockOverviewDataSelector,
+  stockOverviewData => stockOverviewData[LABEL_CURRENT_PRICE]
+);
+
+export const currentPriceValueSelector = createSelector(
+  currentPriceSelector,
+  currentPrice => currentPrice.value
+);
+
+export const previousCloseSelector = createSelector(
+  stockOverviewDataSelector,
+  stockOverviewData => stockOverviewData[LABEL_PREVIOUS_CLOSE]
+);
+
+export const previousCloseValueSelector = createSelector(
+  previousCloseSelector,
+  previousClose => previousClose.value
+);
+
+export function selectedStockValueForKeySelector(state, key) {
+  return createSelector(
+    selectedStockSelector,
+    selectedStock => selectedStock[key]
+  )(state);
+}
+
+export const chartTimePeriodIndexSelector = state => state.chartTimePeriodIndex;
```

Comment:
For organization, maybe group all selectors that aren't dependent on any other selectors together at the top?